api/requirements.in

```diff
@@ -33,3 +34,4 @@ PyYAML>=4.2b1
 
 python-swiftclient
 python-keystoneclient
+
```

Comment:
You should move the test requirements (line 22 until 30 I think) to requirements_dev.in

Reply:
This will require I change the dockerfile and the way the project is deployed. I would like to do this in another PR in the next maintenance.